style.css

```diff
@@ -47,3 +47,24 @@ a:focus {
 a:active {
 	text-decoration: none;
 }
+
+/*
+ * Search and File Block button styles.
+ * Necessary until the following issues are resolved in Gutenberg:
+ * https://github.com/WordPress/gutenberg/issues/36444
+ * https://github.com/WordPress/gutenberg/issues/27760
+ */
+
+.wp-block-search__button,
+.wp-block-file .wp-block-file__button {
+	background-color: var(--wp--preset--color--primary);
+	border-radius: 0;
+	border: none;
+	color: var(--wp--preset--color--background);
```

Comment:
What do you think about adding this line: 

```suggestion
	color: var(--wp--preset--color--background);
	cursor: pointer;
```

Reply:
I recall having a major discussion around that at some point in Core. I think we should leave it to be the default behavior for right now — if that gets changed it should happen in Gutenberg.